src/braket/pulse/pulse_sequence.py

```diff
@@ -44,7 +44,7 @@ class PulseSequence:
 
     def __init__(self):
         self._capture_v0_count = 0
-        self._program = Program()
+        self._program = Program(simplify_constants=False)
```

Comment:
Yes, this is to keep the old behavior. There is a Braket unit test that breaks if I don't specify this. Something gets serialized as `pipipipipi` instead of `5 * pi`. We should find a way to track this as well.